### PR TITLE
Allow symfony 5 in composer.json

### PR DESCRIPTION
**Pull request by wachterjohannes**

### CODE REVIEW

**Comment by nnatter** [comment]
You need to add a xml file for your default template to the test application. For example, see https://github.com/sulu/SuluCommentBundle/pull/38/files#diff-e875b6d0ed122994f5be5c507994abd21e5b56c897a5551880d7e7193139c66bR1

**Comment by nnatter** [comment]
It looks like the pipeline does not install Symfony 5. Maybe there are some additional blocking dependencies?

**Comment by nnatter** [comment]
We need to adjust the tescases to create the client in the `setUp` method. See for example: https://github.com/sulu/SuluRedirectBundle/pull/51/files#

File: composer.json [hunk 1]
```diff
@@ -62,7 +65,7 @@
             "@php-cs"
         ],
         "phpunit": "vendor/bin/phpunit",
-        "phpstan": "vendor/bin/phpstan analyze -c ./phpstan.neon . -l max",
+        "phpstan": "vendor/bin/phpstan analyze -c ./phpstan.neon . -l 5",
```
Comment (nnatter) [inline composer.json:68]: how many errors are there on `max`? 😇

Comment (wachterjohannes) [reply composer.json:68]: 513

Comment (nnatter) [reply composer.json:68]: 😮

File: composer.json [hunk 2]
```diff
@@ -11,31 +11,34 @@
...
+        "phpstan/phpstan-symfony": "^0.12",
+        "phpstan/phpstan-webmozart-assert": "0.12",
         "phpunit/phpunit": "^8.2",
-        "symfony/browser-kit": "^4.3",
-        "symfony/dotenv": "^4.3",
-        "symfony/monolog-bundle": "^3.3",
-        "zendframework/zend-stdlib": "^2.3",
-        "zendframework/zendsearch": "@dev"
+        "symfony/browser-kit": "^4.4 || ^5.0",
+        "symfony/debug": "^4.4 || ^5.0",
```
Comment (nnatter) [inline composer.json:38]: this is deprecated and there is not `5.0` version. but i dont think we are using it anywhere. so i guess we can just remove it?

Comment (nnatter) [reply composer.json]: done this by myself 🙂

**Pull request merged**